examples/storage_profile/aws_with_ebs/README.md

```diff
@@ -0,0 +1,34 @@
+# storage_profile example for AWS with EBS device type
+
+This is an example on how to create an AWS Storage Profile with EBS device type. 
+
+## Getting Started
+
+There are variables which need to be added to terraform.tfvars. The first are for connecting to the vRealize Automation (vRA) endpoint. The others are names of cloud_account, and region that already setup in vRA.
+
+Following is the full list of variables used in this example.
+
+* `url` - The URL for the vRealize Automation (vRA) endpoint
+* `refresh_token` - The refresh token for the vRA account. Alternatively, use the VRA_REFRESH_TOKEN 
+* `cloud_account` - The name of the AWS cloud account added in vRA
+* `region` - The region within the AWS cloud account
+
+To facilitate adding these variables, a sample tfvars file can be copied first:
+```shell
+cp terraform.tfvars.sample terraform.tfvars
+```
+
+Follow these examples for setting up specific cloud accounts:
+
+* Setup [cloud\_account\_aws](../../cloud_account_aws/README.md)
+* Setup [cloud\_account\_azure](../../cloud_account_azure/README.md)
+* Setup [cloud\_account\_gcp](../../cloud_account_gcp/README.md)
+* Setup [cloud\_account\_vmc](../../cloud_account_vmc/README.md)
+* Setup [cloud\_account\_vsphere](../../cloud_account_vsphere/README.md)
```

Comment:
azure link on line 24 says page not found

Do we need all this information for aws storage profile?

Reply:
Done.